framework/CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ Yii Framework 2 Change Log
 
 - Bug #6919: Fixed wrong namespaces under advanced application's TestCase classes (ivokund)
 - Bug #6969: `yii\helpers\ArrayHelper::htmlEncode()` and `htmlDecode()` should not remove non-string data (qiangxue)
+- Bug #6978: DI Container is not reset when destroying application in functional tests (ivokund)
```

Comment:
changelog has to be in https://github.com/yiisoft/yii2/blob/master/extensions/codeception/CHANGELOG.md